### ISSUE TITLE
Add Analytics

### ISSUE THREAD

**Issue by codingwithmanny** [Developer-DAO/developerdao.com#97]
**Context:**
If we ever do promotion for a job board or any marketing it will be worth nothing what the traffic numbers are to use as a reference to pitch companies later down the road.

**Proposal:**
1 - Add Google Analytics code in main website.
OR
2 - Utilize something like Vercel analytics (Just don't know enough about it)

**Requirement:**
The account would need to be opened up by a mod.

**Comment by marbiano** [comment]
Sharing my two cents in here:

- Google Analytics is a decent tool but it's been typically tied to having privacy issues, I don't know if that's a thing or not for a web3 related project.
- Vercel analytics is a free tool only on a `Hobby` plan.
- If there's some kind of budget for the project, my personal preference would be towards using [Fathom](https://usefathom.com), which is a great service and ensures data privacy. Their lowest tier is ~ $12/m if paid yearly.

I don't have enough context to take action on this, but if so, I would love to take it as my first contribution to the repo.

**Comment by codingwithmanny** [comment]
@marbiano 
1 - Agree on the GA privac
2 - We've sponsored by Vercel so this might be included, which would be great, so we would just need to verify. @thomasmetta would you know by chance?
3 - Open to using other platforms as well

The goal of this is just to make sure there's tracking setup so we can just start measuring things as a base, but great feedback on options.

**Comment by marbiano** [comment]
The good thing about Vercel Analytics is that it works out of the box for Next.js projects (no setup required), you just need to go to the projects dashboard, then click on the `Analytics` tab and enable it. Wanna give it a try and let me know if you need any help?

**Comment by codingwithmanny** [comment]
I think @thomasmetta would have to take this one, unless someone else has access.

**Comment by with-heart** [comment]
I have Vercel access as well! Looks like it's not free, unfortunately:

![image](https://user-images.githubusercontent.com/1954752/134172733-ea7e4ebb-ab92-45b5-8ba2-6833a0af3e68.png)

**Comment by thomasmetta** [comment]
Yeah, we can just go with google analytics for now

**Comment by marbiano** [comment]
Happy to take this one, I would need the `GA_TRACKING_ID` in order to integrate it properly.

**Comment by codingwithmanny** [comment]
@thomasmetta we got a standard email address we can set this up with, like a developerdao [at] gmail dotcom or something of the sort?

**Comment by thomasmetta** [comment]
Not yet

**Comment by kempsterrrr** [comment]
@nagmak see above, @marbiano is trying to organise Google Analytics for the website. Could you help organise a GA tracking code with the gmail you set up for the DAO?

**Comment by with-heart** [comment]
I reached out to @nagmak about sharing access to `developerdao1@gmail.com` so hopefully we can get that set up soon!

**Comment by with-heart** [comment]
I have access to the D_D gmail account now!  Do we want to go with GA for analytics?

**Comment by brandonfrulla** [comment]
@with-heart based on the conversation above, I think GA is going to be the temporary solution for now- and down the line we might look at another less-invasive option

**Comment by narbs91** [comment]
I don't have any issues going with Google Analytics, let's do it!

**Comment by emilbryggare** [comment]
I would recommend either Plausible or Fathom. Especially considering that the GA will be blocked by ~50% of visitors. (rough estimate). With the above tools, we can [track data using a proxy](https://plausible.io/docs/proxy/introduction) while still respecting the privacy of our users.

And if I'm not mistaken, we will have to use a cookie consent banner to use Google Analytics. 

Our privacy policy should be updated with relevant information depending on which tracking tool we end up using.

**Comment by codingwithmanny** [comment]
This is true. Privacy policy is going to be crucial also for GDPR.
I'm now voting that be put on hold until we rethink things.

**Comment by narbs91** [comment]
Oh dang that's true around the privacy policy and great that it was brought up, yeah maybe we should huddle on this topic again before committing

**Comment by naomiHauret** [comment]
Like @emilbryggare I would recommend [Plausible](https://plausible.io/) over Google Analytics.
It's very light (~1kb), open-source (here is their [repository](https://github.com/plausible/analytics/)), GDPR compliant and can be self-hosted.

**Comment by JoviDeCroock** [comment]
+1 on Plausible, [Cloudflare](https://www.cloudflare.com/analytics/) has its own analytics as well

**Comment by with-heart** [comment]
Since Plausible has quite a few recommendations (here and on Discord) and also has a strong data privacy policy that appears to cover our needs, I've signed up for an account and will open a PR shortly to connect the site. 

Plausible gives us a 30-day free trial so I'm thinking we see how it goes and we can re-evaluate at the end of the trial?

**Comment by with-heart** [comment]
Just an update on this, I've made our Plausible dashboard stats public! You can see them here: https://plausible.io/developerdao.com
